Sources/Orbit/Components/InputField.swift

```diff
@@ -160,15 +215,45 @@ public extension InputField {
     }
 }
 
+// MARK: - Model
+private class SecureViewModel: ObservableObject {
```

Comment:
Do we need a model + combine for that? 

If we do, it would need to be a `@StateObject` though as @ObervedObject lifetime cannot be handled by the same view itself.

Do we also need to run it every 3 seconds? 

Can we instead use 
```
Task.sleep()
isShowSecuredText
```

directly in the suffix action?

Reply:
> Do we need a model + combine for that?
I fully understand your concern but I failed to do it without the ViewModel. View itself can't change its state, but `isShowSecuredText` has to be switched back to `false`  after some delay (e.g. 3 seconds) to show the secured input

>Do we also need to run it every 3 seconds?
We don't, here we just wait for one event with the help of `.first()` operator and then the publisher finishes.

Task can be used here as well. I've tried it at first, but compiler was complaining about modifying captured self inside. However it shouldn't be the case if the model is a `class`. So I can update it with `Task`

Reply:
I mean - we might not even need to use the Model in the first place (possibly)

Reply:
We need something that will update @State after some delay. I failed to create such mechanism inside of the View because View is not mutable — I can't assign Task, Cancellation later on...